lib/subspace/commands/ssh.rb

```diff
@@ -5,7 +5,8 @@ def initialize(args, options)
   end
 
   def run
-    ansible_command "ansible", @host, '-a', "/bin/bash -l"
+    exec "ssh ubuntu@#{@host}"
```

Comment:
Don't we set this user in one of the config files? Couldn't we do like
`YAML.load(path_to_environments_host_vars_file.yml)["ansible_ssh_user"]`